Source/d_main.c

```diff
@@ -279,7 +279,7 @@ void D_Display (void)
     }
 
   // see if the border needs to be updated to the screen
-  if (gamestate == GS_LEVEL && !automapactive && scaledviewwidth != 320)
+  if (gamestate == GS_LEVEL && (!automapactive || automapoverlay) && scaledviewwidth != SCREENWIDTH)
```

Comment:
Not sure about the `320` -> `SCREENWIDTH` change. :thinking:

Reply:
I revert it back.